packages/fxa-auth-server/lib/subhub/stubAPI.js

```diff
@@ -24,6 +23,56 @@ module.exports.buildStubAPI = function buildStubAPI(log, config) {
     exp_year: 2020,
   };
 
+  const _storeSubscription = uid => subscription => {
```

Comment:
Looks like some good refactoring. The stubs are pretty bad - we were hoping they'd go away soon after real subhub got integrated. But, alas issue #1082 is still a needful thing.